extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,260 @@
+    layer = specular_brdf(α = roughness ^ 2)
+  )
+```
+ to the following:
+```
+dielectric_brdf =
+  fresnel_mix(
+    ior = 1.5,
+    base = mix(
+      diffuse_brdf(color = baseColor),
+      diffuse_btdf(color = diffuseTransmissionColor),
+      diffuseTransmission),
+    layer = specular_brdf(α = roughness ^ 2)
+  )
+```
+Increasing the strength of the diffuse transmission effect using the `diffuseTransmissionFactor` parameter takes away energy from the diffuse reflection BSDF and passes it to the diffuse transmission BSDF. The specular reflection BSDF and Fresnel weighting are not affected.
+
+*This section is non-normative.*
+
+With a simple Lambert BRDF model, `diffuse_brdf` and `diffuse_btdf` may be implemented as follows
+```
+function diffuse_brdf(color) {
+  if (view and light on same hemisphere) {
+    return (1/pi) * color
+  } else {
+    return 0
+  }
+}
+
+function diffuse_btdf(color) {
+  if (view and light on opposite hemispheres) {
+    return (1/pi) * color
+  } else {
+    return 0
+  }
+}
+
+function mix(bsdf0, bsdf1, factor) {
+  return (1-factor) * bsdf0 + factor * bsdf1
+}
+```
+
+<figure style="text-align:center">
+<img src="./figures/bsdf.svg"/>
+<sub><figcaption><em>Left: Diffuse BRDF. Right: Diffuse BTDF.</em></figcaption></sub>
+</figure>
+
+## Combining Diffuse Transmission with other Extensions
+### KHR_materials_transmission
+Both `KHR_materials_diffuse_transmission` and `KHR_materials_transmission` replace the diffuse BRDF with a mix of diffuse BRDF and a BTDF that transmits light onto the opposite side of the surface. In case of `KHR_materials_transmission`, this is a microfacet BTDF that shares its roughness with the microfacet BRDF. In case of `KHR_materials_diffuse_transmission`, on the other hand, this is a diffuse BTDF.
+
+Let's recall the `dielectric_brdf` for `KHR_materials_diffuse_transmission` as defined above
+```
+dielectric_brdf =
+  fresnel_mix(
+    ior = 1.5,
+    base = mix(
+      diffuse_brdf(color = baseColor),
+      diffuse_btdf(color = diffuseTransmissionColor),
+      diffuseTransmission,
+    layer = specular_brdf(α = roughness ^ 2)
+  )
+```
+
+and compare it to the `dielectric_brdf` defined in `KHR_materials_transmission`
+```
+dielectric_brdf =
+  fresnel_mix(
+    ior = 1.5,
+    base = mix(
+      diffuse_brdf(baseColor),
+      specular_btdf(α = roughness^2) * baseColor,
+      transmission),
+    layer = specular_brdf(α = roughness^2)
+  )
+```
+
+Since the diffuse BTDF does not have controls for roughness, the roughness parameter acts only on the reflective part of the surface. By decoupling the reflection and transmission parts it is possible to configure materials which have a smooth reflection and a diffuse transmission, as shown in image below (right).
+
+<figure style="text-align:center">
+<p float="left">
+ <img src="./figures/transmission-translucency_thin.jpg" width="40%" />
+ <img src="./figures/transmission-translucency_thin_angle.jpg" width="39.5%" />
```

Comment:
These images will need to get updated to reflect the new name change to diffuse transmission.